fintech-examples/fintech-db-schema/src/main/resources/migration/migrations/0000-init-schema.xml

```diff
@@ -64,7 +67,24 @@
             </column>
         </createTable>
     </changeSet>
-    <changeSet author="peter (generated)" id="1590439315194-5">
+    <changeSet author="peter (generated)" id="1592477357022-6">
```

Comment:
Do we really need to store this, or maybe it can be read from OPBA. If we need to store this data it should be encrypted